project/DistributionPackage.scala

```diff
@@ -134,6 +134,19 @@ object DistributionPackage {
       distributionRoot / "component",
       cacheFactory.make("engine-jars")
     )
+    val os = System.getProperty("os.name");
+    val parser = if (os.startsWith("Mac")) {
+      "target/rust/debug/libenso_parser.dylib"
+    } else if (os.startsWith("Windows")) {
+      "target/rust/debug/enso_parser.dll"
+    } else {
+      "target/rust/debug/libenso_parser.so"
+    }
```

Comment:
```suggestion
    val prefix = "target" / "rust" / "debug"
    val nativeLibName = if (os.startsWith("Mac")) {
      "libenso_parser.dylib"
    } else if (os.startsWith("Windows")) {
      "enso_parser.dll"
    } else {
      "libenso_parser.so"
    }
    val parser = prefix / nativeLibName
```
something along those lines looks better

Reply:
How can one apply `/` to a string? I am getting `value / is not a member of String` error for:
```scala
val parser = "target" / "rust" / "debug"
```